src/data/firefox-code-coverage-frontend.yaml

```diff
@@ -0,0 +1,36 @@
+name: firefox-code-coverage-frontend
```

Comment:
How about `Firefox Code Coverage Frontend`? it is now like this and I feel like it could be better using space instead of `-`
<img width="377" alt="screen shot 2018-09-29 at 9 46 05 am" src="https://user-images.githubusercontent.com/24544912/46239557-84967e00-c3cc-11e8-8821-16dee43d9b7c.png">